great_expectations/data_context/store/validation_config_store.py

```diff
@@ -25,6 +27,29 @@ def get_key(
             )
         return StringKey(key=name)
 
+    @override
+    @staticmethod
+    def gx_cloud_response_json_to_object_dict(response_json: dict) -> dict:
+        response_data = response_json["data"]
+
+        validation_data: dict
+        if isinstance(response_data, list):
+            if len(response_data) == 0:
```

Comment:
I think we should check for the length being == 1  and treat all other cases as errors. And we should get a test around that